cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -73,6 +98,18 @@ def issue_jwt_token_with_jwk(token_body, file_name, algorithm = Algorithms::RS25
     )
   end
 
+  def issue_none_alg_jwt_token(token_body)
+    token_header_and_payload = JWT.encode(
+      token_body,
+      nil,
+      'none',
+      {
+        kid: jwk_set[@default_file_name].kid
+      }
+    )
+    @jwt_token = "#{token_header_and_payload}invalid_singature"
```

Comment:
`none` signed token does not contain signature part at all
There's no need in extra manipulation.
In all cases output of `JWT.encode` is good to go.